tools/lsp/util.rs

```diff
@@ -71,6 +71,20 @@ pub fn find_element_indent(element: &ElementRc) -> Option<String> {
     None
 }
 
+/// Try to find the parent of element `child` below `root`.
+pub fn search_for_parent_element(root: &ElementRc, child: &ElementRc) -> Option<ElementRc> {
```

Comment:
please avoid adding stuff to that module and keep helper function private near where they are used.
We can always move the function later if they are used by several modules

Reply:
So now the function is there twice, and this one is unused